src/scripts/OSFramework/OSUI/Pattern/Progress/Bar/ProgressBar.ts

```diff
@@ -133,6 +133,10 @@ namespace OSFramework.OSUI.Patterns.Progress.Bar {
 		protected setHtmlElements(): void {
 			// Set the html references that will be used to manage the cssClasses and atribute properties
 			this.progressElem = this.selfElement.querySelector(Constants.Dot + ProgressEnum.CssClass.Container);
+			// Set Progress Circle content element id
+			this.contentElemId = this.selfElement.querySelector(
+				OSUI.Constants.Dot + Progress.ProgressEnum.CssClass.ProgressBarContent
+			).id;
```

Comment:
We should also be setting this to empty on the unsetHtmlElements

Reply:
done!